ydb/core/blobstorage/dsproxy/dsproxy_put_impl.h

```diff
@@ -263,16 +280,19 @@ class TPutImpl {
     void ProcessResponse(TEvBlobStorage::TEvVPutResult& msg) {
         ++VPutResponses;
         ProcessResponseCommonPart(msg.Record);
-        ProcessResponseBlob(VDiskIDFromVDiskID(msg.Record.GetVDiskID()), msg.Record);
+        ui32 orderNumber = Info->GetOrderNumber(VDiskIDFromVDiskID(msg.Record.GetVDiskID()));
+        ProcessResponseBlob(orderNumber, msg.Record);
+        History.AddVPutResult(orderNumber, msg.Record.GetStatus(), msg.Record.GetErrorReason());
     }
 
     void ProcessResponse(TEvBlobStorage::TEvVMultiPutResult& msg) {
         ++VMultiPutResponses;
         ProcessResponseCommonPart(msg.Record);
-        const TVDiskID vdiskId = VDiskIDFromVDiskID(msg.Record.GetVDiskID());
+        ui32 orderNumber = Info->GetOrderNumber(VDiskIDFromVDiskID(msg.Record.GetVDiskID()));
         for (const auto& item : msg.Record.GetItems()) {
-            ProcessResponseBlob(vdiskId, item);
+            ProcessResponseBlob(orderNumber, item);
         }
+        History.AddVPutResult(orderNumber, msg.Record.GetStatus(), msg.Record.GetErrorReason());
```

Comment:
We may have distinct statuses for every item in the EvVMultiPutResult; this Record.GetStatus() does not guarantee they are the same as this one.

Reply:
I don't think we need information about individual statuses to debug performance issues, but it may be useful to know whether the whole request completed successfully or was TIMEOUT'ed.